ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/-components/analyzer-fields.tsx

```diff
@@ -59,12 +61,30 @@ export const AnalyzerFields = ({ form }: AnalyzerFieldsProps) => {
       />
       <AccordionItem value='analyzer' className='flex-1'>
         <AccordionTrigger>Analyzer</AccordionTrigger>
-        <AccordionContent>
+        <AccordionContent className='flex flex-col gap-6'>
+          <FormField
+            control={form.control}
+            name='jobConfigs.analyzer.repositoryConfigPath'
+            render={({ field }) => (
+              <FormItem>
+                <FormLabel>Repository configuration path</FormLabel>
+                <FormControl>
+                  <Input {...field} />
+                </FormControl>
+                <FormDescription>
+                  The optional path to a repository configuration file. If this
+                  is not defined, the repository configuration is read from the
+                  root of the analyzed project repository, ie. "./.ort.yml".
+                </FormDescription>
+                <FormMessage />
+              </FormItem>
```

Comment:
Did some forms use "optional" or something as the placeholder for optional values? If so, this should as well.